### PR TITLE
fix UnicodeDecodeError

### PR DESCRIPTION
**Pull request by areski**
There is a unicodeDecode error on "Rémy" in setup.cfg which prevent the tests passing on https://github.com/mozilla-services/cornice/pull/297

The patch get rid off d2to1

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,9 +1,30 @@
-#!/usr/bin/env python
-from setuptools import setup
+import os
```
Comment (Natim) [inline setup.py]: Maybe we should add a ``# -*- coding: utf-8 -*-`` here?

Comment (areski) [inline setup.py]: Actually this is the first thing I tried before getting rid off d2to1 :)
To be honest I don't know much about "d2to1" but the project seems discontinued

Comment (leplatrem) [inline setup.py]: Thanks a lot @areski for your efforts on this :) 

I would merge like this.

**Comment by Natim** [comment]
Thank you for this, I have tagged a new version with your fix: https://github.com/spiral-project/rxjson/releases/tag/0.3

**Pull request merged**